spec/features/feedback_form_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Feedback form', type: :feature, js: true do
+  before do
```

Comment:
See my comment below about the importance of setting up the stubbed user before loading the page for the "logged in" tests. I think you'll need to put these not logged in tests in a context block so that this before block only applies to these and not the logged in tests.